src/utils/publicMessagesFromGtfs.js

```diff
@@ -15,7 +15,8 @@ function nestGtfs (gtfsRoutes, gtfsAlerts) {
   // filter out alerts that contain routes that we don't recognize
   // can happen if a long-running alert is created for a route before a change in the gtfs file removes it
   gtfsAlerts = gtfsAlerts.filter((gtfsAlert) => {
-    return gtfsAlert.alert.informedEntity.every((entity) => entity.routeId in routesGtfsMap)
+    return gtfsAlert.alert.informedEntity.every((entity) => entity.agencyId === 'SATCo' ||
+                                                            entity.routeId in routesGtfsMap)
```

Comment:
We _technically_ write this app to be agency-agnostic. It'll depend on the structure of the data which I don't have in front of me right now, but we should be able to detect if an alert "applies to no route in particular"?

Maybe something like

```suggestion
    return gtfsAlert.alert.informedEntity.every((entity) => entity.routeId in routesGtfsMap) || gtfsAlert.alert.informedEntity.every((entity) => entity.routeId === undefined)
```

Reply:
Although I'm going to clean that implementation up a little